llvm/lib/MC/MCObjectStreamer.cpp

```diff
@@ -464,6 +464,19 @@ void MCObjectStreamer::emitDwarfAdvanceLineAddr(int64_t LineDelta,
                          Label, PointerSize);
     return;
   }
+
+  // If the two labels are within the same fragment and it's a plain data
```

Comment:
If the two labels are within the same data fragment, then the address-offset is ...

(a data fragment is a MCDataFragment. There is no special or plain data fragment)